### PR TITLE
Unit Test

### PR DESCRIPTION
**Pull request by a2batic**
Current code coverage: 31.72 %
Tendrl-bug-id: #895
Signed-off-by: a2batic <murarkakanika@gmail.com>

### CODE REVIEW

**Comment by gnehapk** [comment]
@a2batic Travis build has failed because some unit tests are not executed successfully. Please correct and amend the PR.

**Comment by a2batic** [comment]
@gnehapk, few test cases are added wrt to #898, I will rebase and update the PR once #898 is merged.

File: test/unit/global-task-list-test.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            expect(vm.isDataLoading).to.be.false;
+            expect(vm.taskDetail.logs).to.deep.equal(globalTaskDetail.logs);
+            expect(vm.isMessagesLoading).to.be.false;
+        });
+
+        /*        it("Should go to cluster detail", function() {
+                    vm.goToClusterDetail();
+                    expect($state.go.calledWith("cluster-hosts", { clusterId: globalTaskDetail.jobDetails.parameters["TendrlContext.integration_id"] })).to.be.true;
+                });WIP
+        */
```
Comment (gnehapk) [inline test/unit/global-task-list-test.js]: @a2batic if this unit test case is not needed, Please remove it.

Comment (a2batic) [reply test/unit/global-task-list-test.js]: Acknowledge

**Review by gnehapk — APPROVED** [review]

**Pull request merged**